3725/2/data/users.yml

```diff
@@ -0,0 +1,73 @@
+members:
```

Comment:
Check this https://symfony.com/doc/current/components/yaml/yaml_format.html#collections